assets/src/hooks/useDeviceSupportsHover.ts

```diff
@@ -0,0 +1,7 @@
+import { useMediaQuery } from "@react-hook/media-query"
```

Comment:
I ended up using the media query hook instead of a purely css solution for 2 reasons:
1. I was running into the pesky "Invalid LatLng" testing user click events for markers when they had both a tooltip and popup, one of which would have been hidden by css. Having only one popup or one tooltip per marker seemed to resolve this issue.
2. It was easier for testing to be able to mock `useDeviceSupportsHover`